src/main/java/com/github/manolo8/darkbot/backpage/FlashResManager.java

```diff
@@ -45,6 +49,8 @@ public void tick() {
                 || currLang.equals(lang)) return;
 
         try {
+            inGameLocale = new Locale(currLang);
```

Comment:
wrap this in a try-catch with the specific exception.
From experience a `new Locale` with a random string (maybe a valid locale in another lang, but not in java) will throw an exception, catch it separately to the generic catch here